xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-test/xwiki-platform-office-test-docker/src/test/it/ui/OfficeImporterIT.java

```diff
@@ -122,103 +124,106 @@ private void deletePageWithChildren(ViewPage pageToDelete)
     /**
      * Delete the page created by the test.
      */
-    private void deletePage()
+    private void deletePage(TestUtils testUtils, String testName)
     {
         DocumentReference pageToDelete =
-            new DocumentReference("xwiki", Arrays.asList(getTestClassName(), getTestMethodName()), "WebHome");
-        getUtil().deletePage(pageToDelete);
+            new DocumentReference("xwiki", Arrays.asList(getClass().getSimpleName(), testName), "WebHome");
+        testUtils.deletePage(pageToDelete);
     }
 
     /**
      * A basic test that imports some documents and verify they are correctly imported TODO: do a more advanced check
      * about styling and content
      */
     @Test
-    public void testImports()
+    public void testImports(TestUtils testUtils)
```

Comment:
No need to prefix with "test". Maybe "verifyImports"? Since we have a single test methods we should find a nice common name.

Reply:
I did not refactor the test style right now. But I can.